packages/core/src/redux/slices/notifications/index.ts

```diff
@@ -157,6 +157,36 @@ startAppListening({
     },
 });
 
+startAppListening({
+    actionCreator: signalEvents.videoEnableRequested,
+    effect: ({ payload }, { dispatch, getState }) => {
+        const { enable, requestedByClientId } = payload;
+
+        const state = getState();
+        const client = selectRemoteParticipants(state).find(({ id }) => id === requestedByClientId);
+
+        if (!client) {
+            console.warn("Could not find remote client that requested a local video change");
+            return;
+        }
+
+        dispatch(
+            doSetNotification(
+                createNotificationEvent<"requestVideoEnable" | "requestVideoDisable", RequestAudioEventProps>({
+                    type: enable ? "requestVideoEnable" : "requestVideoDisable",
+                    message: enable
+                        ? `${client.displayName} has requested for you to start video`
+                        : `${client.displayName} has stopped your video`,
```

Comment:
Should we reflect the message that we show in PWA here. i.e. `${client.displayName} has stopped your video. You can turn your camera back on any time.`. Or perhaps we should just display this message at the demo app toast level and not change this simple messaging here.

Reply:
I was just following the pattern from the mute/unmute notification (which also is different from the PWA). Happy to change it, but then I think we should change both.

Reply:
I don't have a strong opinion on this so let's leave it for now.